### PR TITLE
Redesigned workflow

### PR DESCRIPTION
**Pull request by teymurgahramanov**

### CODE REVIEW

**Review by corest — CHANGES_REQUESTED** [review]
replace all `_` with `-` in action inputs

e.g.
```
artifact_name
```
to
```
artifact-name
```

File: actions/push-image/action.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
+name: build-push-image
```
Comment (corest) [inline actions/push-image/action.yml:1]: split this into two separate actions for was and for azure
for AWS we will use two different accounts for now
and Azure eventually will be removed

But no need to mix everything in the same action

Comment (teymurgahramanov) [reply actions/push-image/action.yml:1]: These two actions are almost the same. I don't see the reason for duplicating.

Comment (teymurgahramanov) [reply actions/push-image/action.yml]: Done

**Pull request closed without merging**